Makefile

```
@@ -11,7 +11,12 @@ export GO111MODULE = on
 
 INCLUDE = -I=. -I=${GOPATH}/src -I=${GOPATH}/src/github.com/gogo/protobuf/protobuf
 BUILD_TAGS?='tendermint'
-BUILD_FLAGS = -mod=readonly -ldflags "-X github.com/tendermint/tendermint/version.GitCommit=`git rev-parse --short=8 HEAD`"
+LD_FLAGS = -X github.com/tendermint/tendermint/version.GitCommit=`git rev-parse --short=8 HEAD`
+BUILD_FLAGS = -mod=readonly -ldflags "$(LD_FLAGS)"
+
+ifeq ($(NOSTRIP),)
+	LD_FLAGS += -s -w
```

```suggestion
  BUILD_FLAGS += -s -w
```

@melekes I reckon it had better initialise BUILD_FLAGS first, then add all the conditional stuff at later stage

---

@alessio if I make it add to build flags then it adds outside the quotation marks so that's why I went with this. 

Unless you mean put "-mod=readonly -ldflags" on the build commands and have the build flags be only `-X github.com/tendermint/tendermint/version.GitCommit=`git rev-parse --short=8 HEAD`
`